src/analysis/special_functions/gamma.lean

```diff
@@ -1214,3 +1215,278 @@ end
+      rw indicator_of_mem,
+      exact ⟨hx, hn⟩ },
+    { simp_rw mul_comm _ (↑x ^ _),
+      refine (tendsto.comp (continuous_of_real.tendsto _) _).const_mul _,
+      convert tendsto_one_plus_div_pow_exp (-x),
+      ext1 n,
+      rw [neg_div, ←sub_eq_add_neg] } },
+  -- let `convert` identify the remaining goals
+  convert tendsto_integral_of_dominated_convergence _ (λ n, (f_ible n).1)
+    (real.Gamma_integral_convergent hs) _
+    ((ae_restrict_iff' measurable_set_Ioi).mpr (ae_of_all _ f_tends)),
+  -- limit of f is the integrand we want
+  { ext1 n,
+    rw [integral_indicator (measurable_set_Ioc : measurable_set (Ioc (_:ℝ) _)),
+      interval_integral.integral_of_le (by positivity: 0 ≤ (n:ℝ)),
+      measure.restrict_restrict_of_subset Ioc_subset_Ioi_self] },
+  -- f is uniformly bounded by the Gamma integrand
+  { intro n,
+    refine (ae_restrict_iff' measurable_set_Ioi).mpr (ae_of_all _ (λ x hx, _)),
+    dsimp only [f],
+    rcases lt_or_le (n:ℝ) x with hxn | hxn,
+    { rw [indicator_of_not_mem (not_mem_Ioc_of_gt hxn), norm_zero,
+        mul_nonneg_iff_right_nonneg_of_pos (exp_pos _)],
+      exact rpow_nonneg_of_nonneg (le_of_lt hx) _ },
+    { rw [indicator_of_mem (mem_Ioc.mpr ⟨hx, hxn⟩), norm_mul, complex.norm_eq_abs,
+        complex.abs_of_nonneg
+          (pow_nonneg (sub_nonneg.mpr $ div_le_one_of_le hxn $ by positivity) _),
+        complex.norm_eq_abs, abs_cpow_eq_rpow_re_of_pos hx, sub_re, one_re,
+        mul_le_mul_right (rpow_pos_of_pos hx _ )],
+      exact one_sub_div_pow_le_exp_neg hxn } }
+end
+
+/-- Euler's limit formula for the Gamma function. -/
+lemma Gamma_seq_tendsto_Gamma (s : ℂ) :
+  tendsto (Gamma_seq s) at_top (𝓝 $ Gamma s) :=
+begin
+  suffices : ∀ (m:ℕ), (-↑m < re s) →
+    tendsto (Gamma_seq s) at_top (𝓝 $ Gamma_aux m s),
+  { rw Gamma,
+    apply this,
+    rw neg_lt,
+    rcases lt_or_le 0 (re s) with hs | hs,
+    { exact (neg_neg_of_pos hs).trans_le (nat.cast_nonneg _), },
+    { refine (nat.lt_floor_add_one _).trans_le _,
+      rw [sub_eq_neg_add, nat.floor_add_one (neg_nonneg.mpr hs), nat.cast_add_one] } },
+  intro m,
+  induction m with m IH generalizing s,
+  { -- Base case: `0 < re s`, so Gamma is given by the integral formula
+    intro hs,
+    rw [nat.cast_zero, neg_zero] at hs,
+    rw [←Gamma_eq_Gamma_aux],
+    { refine tendsto.congr' _ (approx_Gamma_integral_tendsto_Gamma_integral hs),
+      refine (eventually_ne_at_top 0).mp (eventually_of_forall (λ n hn, _)),
+      exact (Gamma_seq_eq_approx_Gamma_integral hs hn).symm },
+    { rwa [nat.cast_zero, neg_lt_zero] } },
+  { -- Induction step: use recurrence formulae in `s` for Gamma and Gamma_seq
+    intro hs,
+    replace hs : -↑m < re (s + 1),
+    { rw [nat.cast_succ, neg_add, ←sub_eq_add_neg, sub_lt_iff_lt_add] at hs,
+      rwa [add_re, one_re] },
+    rw Gamma_aux,
+    have := tendsto.congr' ((eventually_ne_at_top 0).mp
+      (eventually_of_forall (λ n hn, _))) (@tendsto.div_const _ _ _ _ _ _ _ _ s (IH _ hs)),
```

Comment:
This is a strong sign that `div_const` has implicit arguments where it should have explicit ones. Perhaps worth a TODO comment here.

Reply:
After merging master,
```suggestion
      (eventually_of_forall (λ n hn, _))) ((IH _ hs).div_const s),
```